src/web/types.ts

```diff
@@ -39,9 +39,9 @@ declare global {
 }
 
 export type OpenAiProperties<
-  ToolInput extends UnknownObject = {},
+  ToolInput extends UnknownObject = Record<string, never>,
```

Comment:
Use `Record<never, unknown>` in order to fix errors on `src/web/hooks/use-tool-info.test.ts`

```suggestion
  ToolInput extends UnknownObject = Record<never, unknown>,
```